.github/workflows/tests-ces-migrate.yaml

```
@@ -0,0 +1,209 @@
+name: CiliumEndpointSlice migration (ci-ces-migrate)
+
+# Any change in triggers needs to be reflected in the concurrency group.
+on:
+  pull_request: {}
+  push:
+    branches:
+      - main
+      - ft/main/**
+
+permissions: read-all
+
+concurrency:
+  group: ${{ github.workflow }}-${{ github.event.pull_request.number || github.event.after }}
+  cancel-in-progress: true
+
+env:
+  cilium_cli_ci_version:
+  KIND_CONFIG: .github/kind-config.yaml
+
+jobs:
+  check_changes:
+    name: Deduce required tests from code changes
+    runs-on: ubuntu-22.04
+    outputs:
+      tested: ${{ steps.tested-tree.outputs.src }}
+    steps:
+      - name: Checkout code
+        if: ${{ !github.event.pull_request }}
+        uses: actions/checkout@0ad4b8fadaa221de15dcec353f45205ec38ea70b # v4.1.4
+        with:
+          persist-credentials: false
+          fetch-depth: 0
+      - name: Check code changes
+        uses: dorny/paths-filter@de90cc6fb38fc0963ad72b210f1f284cd68cea36 # v3.0.2
+        id: tested-tree
+        with:
+          # For `push` events, compare against the `ref` base branch
+          # For `pull_request` events, this is ignored and will compare against the pull request base branch
+          base: ${{ github.ref }}
+          filters: |
+            src:
+              - '!(test|Documentation)/**'
+
+  setup-and-test:
+    needs: check_changes
+    if: ${{ needs.check_changes.outputs.tested == 'true' }}
+    runs-on: ubuntu-22.04
+    name: Installation and Migration Test
+    strategy:
+      fail-fast: false
+      matrix:
+        include:
+        - name: '1'
+          tunnel: '""'
```

nit
```suggestion
          tunnel: 'disabled'
```
You'd need to modify your `vars` step below, or if you switch to the `cilium-config` action you can just provide the value (ex: https://github.com/cilium/cilium/blob/57db22b2029cd9b725555e0971d9a9f36aad9540/.github/workflows/conformance-e2e.yaml#L304)